test/request.js

```diff
@@ -51,6 +51,27 @@ describe('request', () => {
         assert.equal(httpStub.request.callCount, 1);
     });
 
+    it('should use pathname as path if none specified', () => {
+        const requestStub = new RequestStub();
```

Comment:
The problem is that some test require http (and not https) stub. So this would require larger reorganization, but I agree it's not nice and should be done